using-mobile-extensions/places-extension/places-monitoring-extension/using-your-own-monitor.md

```diff
@@ -1,4 +1,142 @@
 # Using your own monitor
 
-This information is  in progress.
+You can also use your own monitoring services and intergrate with Places using the Places Extension API's
```

Comment:
- Please update this sentence so that it reads as follows:
"You can also use your monitoring services and integrate with Places by using the Places Extension APIs."